storage/mdadm/README.md

```diff
@@ -62,7 +62,7 @@ Now you are ready to create a software raid.
 kubectl -n kube-system debug -it --profile sysadmin --image=alpine node/<node name>
 # you are now in the pod again
 # the following command will create the software raid
-chroot /host mdadm --create /dev/md/<some name> --raid-devices=<amount of devices> --metadata=1.2 --level=<raid level>  <all device paths>
+nsenter --mount=/proc/1/ns/mnt -- mdadm --create /dev/md/<some name> --raid-devices=<amount of devices> --metadata=1.2 --level=<raid level>  <all device paths>
```

Comment:
I actually did that on purpose, rather than expose the complex `nsenter` commands

Reply:
Okay than I will revert this change and fix the command below.